hardware_interface/include/hardware_interface/resource_manager.hpp

```diff
@@ -410,6 +424,12 @@ class HARDWARE_INTERFACE_PUBLIC ResourceManager
   mutable std::recursive_mutex claimed_command_interfaces_lock_;
   mutable std::recursive_mutex resources_lock_;
 
+  std::string interface_values_publisher_name_;
+  std::string interface_values_topic_name_;
```

Comment:
Those two variables are not necessary. At best, we can use constants defined in `.cpp` file for them. But I would just put them in the code.

Reply:
ok